Vic2ToHoI4/Source/Hoi4Outputter/Events/EventsOutputter.h

```diff
@@ -0,0 +1,19 @@
+#ifndef EVENTS_OUTPTUTER_H
```

Comment:
ditto

Reply:
Sigh, sometimes having more eyes retracts from quality. Anyway, happy I caught my error in time.